### ISSUE TITLE
Ability to double-tap to move forward/back n seconds

### ISSUE THREAD

**Issue by autovalue** [halilozercan/BetterVideoPlayer#4]
Kind of like the android YouTube player.

**Comment by tonywamba** [comment]
YouTube doesn't have the swipe left/right to move backward/forward feature. BVP does and in my opinion having the double-tap feature as well will be kinda redundant.

**Comment by autovalue** [comment]
Yes I know, but I was also wondering if it would be possible to disable the swipe functionality (as not everyone wants brightness/volume control via swipes). And then having the double tap as its own feature which can also be disabled if it's not needed.

**Comment by tonywamba** [comment]
I agree with you. It would be great to have that functionality.

**Comment by halilozercan** [comment]
> Yes I know, but I was also wondering if it would be possible to disable the swipe functionality (as not everyone wants brightness/volume control via swipes). And then having the double tap as its own feature which can also be disabled if it's not needed.

I added "disabling swipe" feature recently. It should've been in the library from the start. However, there is still no support for double tap. I think a touch interface would be very cool to have in the library. So that anyone can implement their own touch gestures on top of the video player. BVP can offer 2 such gestures, swipe and double-tap, out-of-box.
